lib/dns/result.js

```diff
@@ -0,0 +1,104 @@
+const packet = require('dns-packet');
+
+function display(r){
+  var header = [r.name, r.ttl, r.class, r.type];
+  var data = Object.values(r.data);
+  var row = header.concat(data);
+  var type;
+  row.unshift("//");
+  switch(r.type){
+    case 'DNSKEY':
+      type = 'base64';
+      break;
+    case 'RRSIG':
+      type = 'base64';
+      break;
+    case 'DS':
+      type = 'hex';
+      break;
+    default:
+      break;
+  }
+  row[row.length -1] = row[row.length -1].toString(type);
+  return row.join("\t");
+}
+
+function buildProofs(results){
+  return results.map((result)=>{
+    let [sigwire, rrdata] = pack(result.rrs, result.sig)
+    let name = result.name;
+    if(name != '.'){ name = name +  '.'; }
```

Comment:
I think you want to check if the name doesn't end with '.', not if it's literally just '.'.